react-components/src/architecture/concrete/observations/ObservationsView.ts

```diff
@@ -0,0 +1,161 @@
+import {
+  CDF_TO_VIEWER_TRANSFORMATION,
+  Overlay3DCollection,
+  type OverlayInfo,
+  type CustomObjectIntersectInput,
+  type CustomObjectIntersection
+} from '@cognite/reveal';
+import { type DomainObjectIntersection } from '../../base/domainObjectsHelpers/DomainObjectIntersection';
+import { Changes } from '../../base/domainObjectsHelpers/Changes';
+import { type DomainObjectChange } from '../../base/domainObjectsHelpers/DomainObjectChange';
+import { createObservationIntersection, type Observation } from './types';
+import { ClosestGeometryFinder } from '../../base/utilities/geometry/ClosestGeometryFinder';
+import { getColorFromStatus } from './color';
+
+type ObservationCollection = Overlay3DCollection<Observation>;
+
+export class ObservationsView extends GroupThreeView<ObservationsDomainObject> {
+  private readonly _overlayCollection: ObservationCollection = new Overlay3DCollection([]);
+
+  protected override calculateBoundingBox(): Box3 {
+    return this._overlayCollection
+      .getOverlays()
+      .reduce(
+        (box, overlay) => (overlay.getVisible() ? box.expandByPoint(overlay.getPosition()) : box),
+        new Box3()
+      );
+  }
+
+  protected override addChildren(): void {
+    const observations = this.domainObject.getObservations();
+
+    const selectedObservation = this.domainObject.getSelectedObservation();
+    const overlayInfos = createObservationOverlays(observations, selectedObservation);
+    this._overlayCollection.removeAllOverlays();
+
+    this._overlayCollection.addOverlays(overlayInfos);
+    this.updateClipping();
+
+    this.addChild(this._overlayCollection);
+  }
+
+  public override update(change: DomainObjectChange): void {
+    super.update(change);
+
+    if (change.isChanged(Changes.geometry)) {
+      this.clearMemory();
+      this.invalidateRenderTarget();
+      this.invalidateBoundingBox();
+    } else if (change.isChanged(Changes.clipping)) {
+      this.updateClipping();
+    } else if (change.isChanged(Changes.selected)) {
+      this.updateColors();
+      this.invalidateRenderTarget();
+    }
+  }
+
+  public override intersectIfCloser(
+    intersectInput: CustomObjectIntersectInput,
+    closestDistance: number | undefined
+  ): undefined | CustomObjectIntersection {
+    const { domainObject } = this;
+
+    const closestFinder = new ClosestGeometryFinder<DomainObjectIntersection>(
+      intersectInput.raycaster.ray.origin
+    );
+
+    if (closestDistance !== undefined) {
+      closestFinder.minDistance = closestDistance;
+    }
+
+    const intersectedOverlay = this._overlayCollection.intersectOverlays(
+      intersectInput.normalizedCoords,
+      intersectInput.camera
+    );
+
+    if (intersectedOverlay === undefined || !intersectedOverlay.getVisible()) {
+      return undefined;
+    }
+
+    const point = intersectedOverlay.getPosition();
+
+    if (domainObject.useClippingInIntersection && !intersectInput.isVisible(point)) {
```

Comment:
You do not need this statement since you already have updated by the clipping planes.